File: School source codes/Lab 5/Checkpoint.cs
using System;
using System.Linq;

class MainClass {
  public static void Main (string[] args) {
    int xvalue,yvalue;
    Console.Write("X coordinate is: ");
    xvalue = Convert.ToInt32(Console.ReadLine());
    Console.Write("Y coordinate is: ");
    yvalue = Convert.ToInt32(Console.ReadLine());
    if (xvalue == 0) {
      if (yvalue == 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is the origin point");}
      else
        {Console.WriteLine("("+xvalue+","+yvalue+") is on the Y Axis");}}
    else if (xvalue > 0) {
      if (yvalue == 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is on the X Axis");}
      else if (yvalue > 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is in the first quadrant");}
      else if (yvalue < 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is in the fourth quadrant");}}
    else {
      if (yvalue == 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is on the X Axis");}
      else if (yvalue > 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is in the second quadrant");}
      else if (yvalue < 0)
        {Console.WriteLine("("+xvalue+","+yvalue+") is in the third quadrant");}}
  }
}

File: School source codes/Lab 5/Checkpoint.rb
puts "X coordinate is: "
xvalue = gets.chomp.to_i
puts "Y coordinate is: "
yvalue = gets.chomp.to_i
if (xvalue == 0) 
  if (yvalue == 0)
    puts "(#{xvalue},#{yvalue}) is the origin point"
  else
    puts "(#{xvalue},#{yvalue}) is on the Y Axis"
  end
elsif (xvalue > 0) 
  if (yvalue == 0)
    puts "(#{xvalue},#{yvalue}) is on the X Axis"
  elsif (yvalue > 0)
    puts "(#{xvalue},#{yvalue}) is in the first quadrant"
  else
    puts "(#{xvalue},#{yvalue}) is in the fourth quadrant"
  end
else
  if (yvalue == 0)
    puts "(#{xvalue},#{yvalue}) is on the X Axis"
  elsif (yvalue > 0)
    puts "(#{xvalue},#{yvalue}) is in the second quadrant"
  else 
    puts "(#{xvalue},#{yvalue}) is in the third quadrant"
  end
end